Sources/App/Models/SocialAccount/Social.swift

```diff
@@ -0,0 +1,92 @@
+import Vapor
+import FluentProvider
+import HTTP
+
+final class Social: Model {
+  let storage = Storage()
+  
+  var socialId: Int
+  var name: String
+  var appId: String
+  var secureKey: String
+  var serviceToken: String
+  
+  struct Keys {
+    static let socialId = "social_id"
```

Comment:
We talked with @smejl about name convention for PK, and decided that in DB all primary keys should be called `id`, not `social_id`, etc. So I think you can rename it to `id`. @smejl do you confirm it?

Reply:
Id is by default, isn't it?

Reply:
Id of object I guess.

Reply:
Yes, plus it's by default in `storage` variable, but we still need to add it to `Keys` for referring without string literals

Reply:
@dannydaddy3 If I have understood you correctly, you suggest to make as in SocialAccount: ``` struct Keys {
    static let id = "id"
}```
But in SocialAccount FK will be the same as now. Right?

Reply:
Yes, exactly, so PK is always id(and you shouldnt add it in model as property, cuz its already in `storage` property, only to Keys struct), and FK is entityId